runtime/service.go

```diff
@@ -681,6 +668,33 @@ func (s *service) buildVMConfiguration(req *proto.CreateVMRequest) (*firecracker
 	return &cfg, nil
 }
 
+func (s *service) buildNonStubDrives(req *proto.CreateVMRequest) []models.Drive {
+	var builder firecracker.DrivesBuilder
+
+	if input := req.RootDrive; input != nil {
+		builder = firecracker.DrivesBuilder{}.
```

Comment:
We don't need to create a new builder since we could use the one above.

```go
builder = builder.WithRootDrive(...)
```

Reply:
Ah, right. Still doing Java here. Addressed at https://github.com/firecracker-microvm/firecracker-containerd/pull/283/commits/96c7417c7fe06a4d8f2c25150c83010d9e6e400c